src/Spago/Glob.js

```diff
@@ -1,4 +1,5 @@
 import mm from 'micromatch';
+import picomatch from 'picomatch';
```

Comment:
Now that we import explicitly from `picomatch` we should add it as a dependency to the `package.json`

Reply:
I'll just merge it in and add this while I release 🙂